app.js

```diff
@@ -560,6 +561,25 @@ var eventsModule = function () {
             //   values (CSV), into a hierarchy using d3.stratify.
             var root = EFsToHierarchy();
 
+            var efDurationSum = function(efNode) {
+                if (efNode.children === undefined) {
+                    // Node has no children, return duration value
+                    return efNode.data.durationMinutes;
+                } else {
+                    // Node has children, compute duration of each child
+                    var childSum = 0;
+                    for (var i = 0; i < efNode.children.length; i++) {
+                        childSum += efDurationSum(efNode.children[i]);
+                    }
+                    return childSum;
+                }
+            }
```

Comment:
Now that i think about it, i think this summation of the durations may be better to implement in the `eachBefore` function on [line 433](https://github.com/jrsconfitto/VisVirtualHackathon/pull/51/files#diff-0364f57fbff2fabbe941ed20c328ef1aR433). That's where the cells are processed, one at a time, to fit into the tree-like data structure.

i suggest moving it there.